### ISSUE TITLE
parse --include-comments and build back issue 

### ISSUE THREAD

**Issue by Taymindis** [nginxinc/crossplane#57]
I am having a issue when parse --include comments and build back issue. the comment sign will also comment out some of line

for example :-
it comments out my `user root;`
```nginx
# This config was built from JSON using NGINX crossplane.# If you encounter any bugs please report them here:# https://github.com/nginxinc/crossplane/issuesuser root;
```

before parse is 
```nginx
# This config was built from JSON using NGINX crossplane.
# If you encounter any bugs please report them here:
# https://github.com/nginxinc/crossplane/issues

user root;
```

**Comment by aluttik** [comment]
Nice find! I looked into it and apparently crossplane is having problems building when a file is supposed to start with comments. I'll fix that and notify you when the new patch version is uploaded to PyPI.

**Comment by aluttik** [comment]
The fix for this has been merged and will be included in 0.4.15
